docs/examples/native_asset.md

```diff
@@ -0,0 +1,305 @@
+## Get Balance
+
+### with NEO-Scan API
+
+```javascript
+const { default: Neon, api } = require("@cityofzion/neon-js");
+
+apiProvider
+ .getBalance(myWallet.accounts[1].address)
+ .then(res =>
+    console.log(
+      `NEO: ${res.assets["NEO"].balance.toNumber()},
+       GAS: ${res.assets["GAS"].balance.toNumber()}`
+   )
+ )
+ .catch("Get Balance Error!");
+```
+
+### with RPC Query
+
+```javascript
+const { rpc } = require("@cityofzion/neon-js");
+
+rpc.Query.getAccountState(myWallet.accounts[1].address)
+  .execute("http://rpc.url:portNum")
+  .then(response => {
+    console.log(
+      `NEO: ${response.result.balances[0].value},
+       GAS: ${response.result.balances[1].value}`
+    );
+  })
+  .catch("Get Balance Error!");
+```
+
+
+## Send NEO & GAS
+
+### with Neon API (High Level)
+
+```javascript
+const { default: Neon, api } = require("@cityofzion/neon-js");
+
+// Receiver address
+const receivingAddress = "AJdNhG7qh6p2CNxMnEyb5xGrER5HmsxwjH";
+const intent = api.makeIntent({ NEO: 1, GAS: 1 }, receivingAddress);
+ 
+// Sender configs
+const config = {
+  api: apiProvider, // The network to perform the action, MainNet or TestNet.
+  url: "http://rpc.url:portNum",
+  account: myAccount, // This is the address which the assets come from.
+  intents: intent, // This is where you want to send assets to.
+  gas: 0, // Optional, this is additional gas paying to system fee.
+  fees: 0 // Optional, additional gas paying for network fee(prioritizing, oversize transaction).
+};
+ 
+// Neon API
+Neon.sendAsset(config)
+ .then(config => {
+    console.log("\n\n--- Response ---");
+    console.log(config.response);
+ })
+ .catch(config => {
+    console.log(config);
+ });
+```
+
+
+
+### With NEO-Scan API (Low Level)
+
+```javascript
+const { default: Neon, rpc, api } = require("@cityofzion/neon-js");
+
+const receivingAddress = "AaEvSJVCD3yvoWYR75fLwNutmDKKUzaV6w";
+ 
+// create contract transaction using NEO-Scan API
+async function createTxWithNeoScan() {
+  let balance = await apiProvider.getBalance(myAccount.address);
+  let transaction = Neon.create.contractTx();
+  transaction
+    .addIntent("NEO", 1, receivingAddress)
+    .calculate(balance)
+    .sign(myAccount.privateKey);
+ 
+  return transaction;
+}
+ 
+// Send raw transaction
+const client = new rpc.RPCClient("http://rpc.url:portNum");
+createTxWithNeoScan().then(transaction => {
+  console.log(transaction);
+  client
+    .sendRawTransaction(transaction)
+    .then(res => {
+      console.log(res);
+    })
+    .catch(err => {
+      console.log(err);
+    });
+});
+```
+
+
+
+### Constructing Raw Transaction
+
+This method is not recommended in neon-js.
+
+```javascript
+const { default: Neon, wallet, rpc, tx } = require("@cityofzion/neon-js");
+
+// Receiver address and script hash
+const receivingAddress = "AT27F9e1HaUHi6LNhxafVFMDrajtWVksNq";
+const receivingAddressScriptHash = "7b5a8ec29318ad44143a29f947357995824f2ecc";
```

Comment:
You should still be using the Account object instead of manually deriving the scripthash.